spacemesh/v1/admin_types.proto

```diff
@@ -79,15 +84,35 @@ message EventPostComplete {
   bytes smesher = 2;
 }
 
+// Deprecated. Will be removed soon. Use WaitingForPoETRegistrationWindow instead.
 message EventPoetWaitRound {
+  option deprecated = true;
   uint32 current = 1;
   uint32 publish = 2;
   google.protobuf.Duration wait = 3;
   google.protobuf.Timestamp until = 4;
   bytes smesher = 5;
 }
 
+message EventWaitingForPoETRegistrationWindow {
+  uint32 current = 1;
+  uint32 publish = 2;
+  google.protobuf.Duration wait = 3;
+  google.protobuf.Timestamp until = 4;
+  bytes smesher = 5;
+}
+
+// Deprecated. Will be removed soon. Use EventWaitingForPoETRoundEnd instead.
 message EventPoetWaitProof {
+  option deprecated = true;
+  uint32 publish = 1;
+  uint32 target = 2;
+  google.protobuf.Duration wait = 3;
+  google.protobuf.Timestamp until = 4;
+  bytes smesher = 5;
+}
+
+message EventWaitingForPoETRoundEnd {
   uint32 publish = 1;
   uint32 target = 2;
   google.protobuf.Duration wait = 3;
```

Comment:
These two differ only in their name. Perhaps it would be better to keep the old one (less complication in the API).

Reply:
The whole point there was to do the rename.
Having that duplicated leaves "old" untouched and simpler to remove in the future.
Keeping only one of them there would make emitters code in node `events/events.go` less readable IMO.

Reply:
@jellonek 
>Keeping only one of them there would make emitters code in node events/events.go less readable IMO.

I don't understand. Do you mean the name `EventPoetWaitProof` is less readable than `EventWaitingForPoETRoundEnd`?

Reply:
It a bit more precisely describes what happening in that part of the code.

Reply:
What about the perspective of the consumer of this event (the user)? The user knows the node should wait for poet _proof_.